install/kubernetes/tetragon/values.yaml

```diff
@@ -395,3 +395,6 @@ rthooks:
     override: ~
     repository: quay.io/cilium/tetragon-rthooks
     tag: v0.3
+  # -- rthooks service account.
+  serviceAccount:
```

Comment:
IIRC in the previous version the default was the release name, [like for the agent](https://github.com/cilium/tetragon/blob/main/install/kubernetes/tetragon/templates/_helpers.tpl#L69-L75).
With the new version you don't get the credentials of the tetragon SA to access the API. An alternative approach to achieve the same is with adding `automountServiceAccountToken: false` to spec.template.spec of the rthooks daemonset. This may be very specific to OpenShift but the SA identity will be checked for admitting a  pod using hostpath. Both the agent and the rthooks daemonset need it, which means that their respective SAs need to be bound to a convenient SCC.
The answer may be documentation but I wanted to mention this point.

Reply:
> With the new version you don't get the credentials of the tetragon SA to access the API

API here being the k8s API server, correct? I think that's fine since rthooks does not need access to the API server. I think I'll just merge the change for now. Once/if we have a concrete problem, it would be easier to figure it out then.

Thanks!